### DISCUSSION TITLE
The nav bar's color of the documentation need to be updated !!

### DISCUSSION THREAD

**Discussion by mahadi-hossain** [apache/pekko#214] [Ideas]
I think the color of the nav bar and section links of https://pekko.apache.org/docs/pekko/current/ need to be updated. 
Keeping web site color adjusted with logo will give more professional look.
The update can be like the below one.
![Screenshot from 2023-02-22 00-00-04](https://user-images.githubusercontent.com/13967445/220424373-83f61bbc-54ad-4e3a-8e6e-710e28f71da9.png)

**Comment by jrudolph** [comment]
Thanks for the observation. We didn't do any customization of colors so far, indeed. Do you mean the contrast between logo and bar is not visually pleasing?

![image](https://user-images.githubusercontent.com/9868/221808267-1bb95b2f-8304-45c4-b85e-ede622d92932.png)

Do you mean we should also update the general color scheme to adapt logo colors?

**Reply by mahadi-hossain** [reply]
Yes, you are right.
